src/PhotoGallery/AsyncDownloadPhotoTrigger.cc

```diff
@@ -0,0 +1,146 @@
+#include "AsyncDownloadPhotoTrigger.h"
```

Comment:
There should be the copyright header in all these new files.

Reply:
Added to all files.